src/views/OpenWeatherAPIPage.vue

```diff
@@ -56,7 +60,7 @@ export default class ApiClass extends Vue {
   getTheWeather() {
     const url = 'https://api.openweathermap.org/data/2.5/forecast?q='.concat(this.city)
       .concat('&units=imperial')
-      .concat('&appid=').concat(process.env.VUE_APP_API_ACCESS_KEY);
+      .concat('&appid=').concat(process.env.VUE_APP_API_ACCESS_KEY!);
```

Comment:
For the curious...
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Basically this makes the static type checker happy since it has no way to know if the process.env variable is set prior to runtime.

Reply:
... I was just going to ask about that! Thanks for the info